app/helpers/supplier_helper.rb

```diff
@@ -2,6 +2,6 @@
 
 module SupplierHelper
   def supplier_link(supplier)
-    supplier.is_a?(LeadProvider) ? nil : admin_delivery_partner_path(supplier)
+    supplier.is_a?(LeadProvider) ? nil : edit_admin_delivery_partner_path(supplier)
```

Comment:
Just noticed this, what happens when the value for govuk link is nil?

Reply:
Link to the same page. Will be removing the LP links in a separate PR though